### PR TITLE
chore: bump Dockerfile SERVER_VERSION to 0.12.0

### PR DESCRIPTION
**Pull request by simon824**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,16 +19,16 @@ RUN set -x \
 # 2. Init HugeGraph Sever
 # (Optional) You can set the ip of github to speed up the local build
 # && echo "192.30.253.112 github.com\n151.101.44.249 github.global.ssl.fastly.net" >> /etc/hosts \
-ENV SERVER_VERSION 0.10.4
+ENV SERVER_VERSION 0.12.0
 RUN set -e \
     && mkdir -p /root/hugegraph-server \
-    && curl -L -S ${PKG_URL}/hugegraph/releases/download/v${SERVER_VERSION}/hugegraph-${SERVER_VERSION}.tar.gz -o /root/server.tar.gz \
+    && curl -L -S ${PKG_URL}/incubator-hugegraph/releases/download/v${SERVER_VERSION}/hugegraph-${SERVER_VERSION}.tar.gz -o /root/server.tar.gz \
```
Comment (javeme) [inline Dockerfile:25]: we can keep hugegraph url, since hugegraph will be redirected to incubator-hugegraph

Comment (simon824) [reply Dockerfile]: make sense

**Comment by imbajin** [comment]
Thanks, a few questions:
1. After the upgrade, do we test initialization and startup is OK?
2. Could studio connect to server 0.12? (and maybe we could use hubble instead?)

Links the TODO issue

File: Dockerfile [hunk 2]
```diff
@@ -19,16 +19,16 @@ RUN set -x \
...
     && mkdir -p /root/hugegraph-server \
     && curl -L -S ${PKG_URL}/hugegraph/releases/download/v${SERVER_VERSION}/hugegraph-${SERVER_VERSION}.tar.gz -o /root/server.tar.gz \
     && tar xzf /root/server.tar.gz --strip-components 1 -C /root/hugegraph-server \
     && rm /root/server.tar.gz \
     && cd /root/hugegraph-server/ \
     && sed -i "s/^restserver.url.*$/restserver.url=http:\/\/0.0.0.0:8080/g" ./conf/rest-server.properties \
-    && sed -n '63p' ./bin/start-hugegraph.sh | grep "&" > /dev/null && sed -i 63{s/\&$/#/g} ./bin/start-hugegraph.sh \
-    && sed -n '74p' ./bin/start-hugegraph.sh | grep "exit" > /dev/null && sed -i 74{s/^/#/g} ./bin/start-hugegraph.sh \
+    && sed -n '65p' ./bin/start-hugegraph.sh | grep "&" > /dev/null && sed -i 65{s/\&$/#/g} ./bin/start-hugegraph.sh \
+    && sed -n '75p' ./bin/start-hugegraph.sh | grep "exit" > /dev/null && sed -i 75{s/^/#/g} ./bin/start-hugegraph.sh \
```
Comment (imbajin) [inline Dockerfile:31]: TODO:  we could add a option `-f` to run foreground (to avoid modify shell)

**Comment by simon824** [comment]
> Thanks, a few questions:
> 
> 1. After the upgrade, do we test server initialization and startup OK?
> 2. Could studio connect to server 0.12? (shall we upgrade the version or use hubble instead?)
> 
> Links the TODO issue, and we need upload the image to Dockerhub

Maybe we can remove `studio` here and add another separate dockerfile for `hubble` ？ WDYT @javeme @imbajin

**Comment by simon824** [comment]
BTW, i find `apache` has it's own docker repository https://hub.docker.com/u/apache, do we need move https://hub.docker.com/u/hugegraph to apache's ?

**Comment by imbajin** [comment]
> BTW, i find `apache` has it's own docker repository [hub.docker.com/u/apache](https://hub.docker.com/u/apache), do we need move [hub.docker.com/u/hugegraph](https://hub.docker.com/u/hugegraph) to apache's ?

fine if it's easy to move

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**